homes_guests_loves.html

```diff
@@ -73,14 +76,16 @@ <h1 class="col-lg-12 col-sm-8">The best destinations</h1>
             <p class="home-hotel-name home-text">Island</p>
 
         </div>
-        <form>
-            <button class="best-destinations-book">Book now</button>
-        </form>
+<!--        <form>-->
+<!--            <button class="best-destinations-book">Book now</button>-->
+<!--        </form>-->
```

Comment:
there should be no comments when you push updates

Reply:
done